src/tests/SimTest.cpp

```diff
@@ -124,8 +164,116 @@ void SimTest::getPinholeCameraRGBAObservation() {
           Mn::PixelFormat::RGBA8Unorm,
           {pinholeCameraSpec->resolution[0], pinholeCameraSpec->resolution[1]},
           observation.buffer->data}),
-      Cr::Utility::Directory::join(TEST_DIR, "SimTestExpected.png"),
-      (Mn::DebugTools::CompareImageToFile{15.0f, 0.17f}));
+      Cr::Utility::Directory::join(screenshotDir, groundTruthImageFile),
+      (Mn::DebugTools::CompareImageToFile{30.f, 0.5f}));
```

Comment:
Not sure if having a global comparison threshold for all tests is a good idea in the long run -- I can imagine some particular test scenes might exhibit larger differences across drivers than others, and then globally increasing the thresholds would give a false sense of security.

Maybe have the min/mean threshold as a parameter to `checkPinholeCameraRGBAObservation()` instead?

Reply:
looking at the test failures there seem to be massive differences across drivers (a few pixels vary by over 100). The good news is that mean differences are still very low. Do you think we should just ignore individual pixel differences and only test mean?

Reply:
@aclegg3 @mosra @bigbike the other option would be to only run these visual tests in CI, but this would be annoying for debugging build failures. Is anyone opposed to only checking the mean? Lets try to get this solved soon since this seems to be the only blocker.

Reply:
I am not opposed to just checking the mean.  I view these tests as more as smoke tests than anything (are the pipes all still hooked up correctly) -- to that end, if you try to break things, does the mean catch it? i.e. turn off phong shading for one that test that uses it, change the light color dramatically, etc.

Reply:
Those massive differences in few pixels are expected, and are due to differences in how rasterizers work on various GPUs. The mean will still catch it I think.

That said, I'm open to ideas how to handle this better in the image comparison tool. Some options I was thinking of:

1. removing (some configurable percentage of) largest outliers, checking max/mean only in the rest
2. applying a slight gaussian blur before doing the comparison which should not cause false positives since it's IIRC a reversible operation, but on the other hand should account better for slight spatial differences and differences in edge rasterization
3. something else? (how do browser test frameworks do it? any obvious tricks I'm missing?)

Reply:
I think most frameworks use software rendering for exact image comparison tests.  I do like the idea of outlier removal.  Unlikely that a few pixels being very wrong is actually a problem.

Reply:
Great. I'll return with a PR that adapts the test when the outlier removal is implemented.